File: Models/ModeliAplikacije/KorisnikKarta.cs
﻿using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace AutobuskaStanicaInternetProgramiranje.Models.ModeliAplikacije
{
    public class KorisnikKarta
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public string KorisnikID { get; set; }
        public int KartaID { get; set; }
    }
}

File: ClientApp/src/models/korisnikKarta.ts
import { karta } from "./karta";
import { korisnik } from "./korisnik";

export interface korisnikKarta {
  id: number;
  korisnikID: number;
  kartaID: number;
  //korisnik?: korisnik; 
  //karta?: karta;
}
